### PR TITLE
fixes #736

### LINKED ISSUE #736: [BUG] Player Info Icon in Squad column not visible
Player info Icons in Squad column are completly white in skins

When I use skins
Nimbus
Darcula 
Solarized
Solarized white

The Player info Icons are just white blocks in column "Squad" and in the Player Information Pull Down Menu.

**Screenshots**
https://ibb.co/bvB0bzb
HO40r3266 skin solarized 2020-10-19 2138.jpg

Old Version to compare:
https://ibb.co/NW0d3f5
HO30 skin nimbus 2020-10-19 2159.jpg

**Platform information:**
 - OS: Windows 10 Pro 64-bit Version 2004 Build 19041.572
 - Version HO! 4.0 r3266 
**Additional context**
Is it possible for users to change colors in skins? I like to change background colour of Squad letters as well, "F" is way to bright!

### PR DESCRIPTION
**Pull request by akasolace**
1. changes proposed in this pull request:
 
   - fixes issue #736
   
   - if not fixing an existing issue, comments explaining the purpose of the PR should be provided)
 
 
2. changelog and release_notes ...

 - [ ] have been updated
 - [x] do not require update

### CODE REVIEW

File: src/main/java/core/gui/theme/ho/HOClassicSchema.java [hunk 1]
```diff
@@ -43,15 +43,11 @@ private void initIcons() {
...
-		put(MANUELLSMILIES[5], "gui/bilder/smilies/5verysad.gif");
-		put(MANUELLSMILIES[6], "gui/bilder/smilies/6clown.gif");
-		put(MANUELLSMILIES[7], "gui/bilder/smilies/7trainer.gif");
-		put(MANUELLSMILIES[8], "gui/bilder/smilies/8dollar.gif");
-		put(MANUELLSMILIES[9], "gui/bilder/smilies/9coach.gif");
+		put(SMILEYS[1], "gui/bilder/smilies/smiley-coach.svg");
+		put(SMILEYS[2], "gui/bilder/smilies/smiley-sale.svg");
+		put(SMILEYS[3], "gui/bilder/smilies/smiley-happy.svg");
+		put(SMILEYS[4], "gui/bilder/smilies/smiley-sad.svg");
+		put(SMILEYS[5], "gui/bilder/smilies/smiley-neutral.svg");
```
Comment (tychobrailleur) [inline src/main/java/core/gui/theme/ho/HOClassicSchema.java:50]: There are 5 icons now, as opposed to 9 before, is this intentional?

File: src/main/java/module/tsforecast/TSPanel.java [hunk 2]
```diff
@@ -82,7 +82,7 @@
   public static final Color lightYellow = new Color(255, 255, 200);
 
   private final Icon m_newImage   = ThemeManager.getIcon(HOIconName.MATCHICONS[2]);
-  private final Icon m_startImage = ThemeManager.getIcon(HOIconName.MANUELLSMILIES[7]);
+  private final Icon m_startImage = ThemeManager.getIcon(HOIconName.SMILEYS[7]);
```
Comment (tychobrailleur) [inline src/main/java/module/tsforecast/TSPanel.java:85]: There are only 6 elements in the array, does this not cause a problem?

File: src/main/resources/gui/bilder/smilies/smiley-coach.svg [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (tychobrailleur) [inline src/main/resources/gui/bilder/smilies/smiley-coach.svg:57]: This probably could be cleaned up.

File: src/main/java/module/tsforecast/TSPanel.java (hunk 2 above)
Comment (akasolace) [reply src/main/java/module/tsforecast/TSPanel.java:85]: good catch !

File: src/main/java/core/gui/theme/ho/HOClassicSchema.java (hunk 1 above)
Comment (akasolace) [reply src/main/java/core/gui/theme/ho/HOClassicSchema.java:50]: wanted to simplify it a bit. Also was lazy founding all icons in svg format ...

**Comment by akasolace** [comment]
@tychobrailleur I made the required adjustments, should be ok for merge now ....

**Pull request merged**